src/libraries/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -1311,7 +1311,7 @@ public static TSource ElementAt<TSource>(this IQueryable<TSource> source, Index
         }
 
         [DynamicDependency("DefaultIfEmpty`1", typeof(Enumerable))]
-        public static IQueryable<TSource> DefaultIfEmpty<TSource>(this IQueryable<TSource> source)
+        public static IQueryable<TSource?> DefaultIfEmpty<TSource>(this IQueryable<TSource> source)
```

Comment:
Please update the signature in the reference source as well:

https://github.com/dotnet/runtime/blob/5d3288d449ed61fa1e9da3fe876400ac14c47fa7/src/libraries/System.Linq.Queryable/ref/System.Linq.Queryable.cs#L78

Reply:
Updated in https://github.com/kanadaj/runtime/commit/9e9e6dae6cd3dc3605ed199fd9b9f879bd7a0326, thanks for the heads-up!